### ISSUE TITLE
[RFE] add output of nodetool status to the monitoring.

### ISSUE THREAD

**Issue by Pawel-Kr** [scylladb/scylla-monitoring#1590]
**System information**
- Scylla version (you are using): n/a
- Are you willing to contribute it (Yes/No): no

**Describe the feature and the current behavior/state.**
It'd be very convenient to have output of `nodetool status` command available to see in the monitoring. This way it will be possible to see if the node is really down, or is it just overloaded and it's firing false alerts. Also, for smaller clusters, it'll allow to check if the data is evenly distributed (in cases when commitlog is much bigger than actual data on the disk).

**Who will benefit with this feature?**
Everyone

**Any Other info**
n/a

**Comment by amnonh** [comment]
The node status (i.e. joining, leaving, normal etc') is already part of the nodes table

**Comment by Pawel-Kr** [comment]
That's right, but `nodetool status` is having much more useful info, for example it will be possible to track nodes, from which rack, are down. So for example if 2/3 nodes from one rack are down, then we know it immediately without need to log into the cluster. Also load indicators are much more precise than disk usage under OS metrics dashboard. @tarzanek probably will point a few other things also.

**Comment by amnonh** [comment]
@Pawel-Kr That's why I didn't close this issue and only added the relevant information

**Comment by amnonh** [comment]
relates to https://github.com/scylladb/scylla/issues/9737
relates to https://github.com/scylladb/scylla/issues/9749

**Comment by amnonh** [comment]
@tarzanek what are the other things you find useful and that are missing?

**Comment by tarzanek** [comment]
@amnonh thanks for filing https://github.com/scylladb/scylla/issues/9737 
I thought scylla already reports that, you have to still take that metric "easy", but I agree it's a good addition in comparison to the disk reports from node exporter - and I definitely agree with adding it

would hostid report be useful too? (e.g. for backups it definitely will, but this data is also in metadata of backup, so do we want to duplicate it? maybe to see over time which ids were added and which removed? hmm, I don't think it's important)
dtto for tokens (likely no one changes them anyways)

**Comment by amnonh** [comment]
We already report and show the load by table, closing, re-open with concrete information if something is missing
